{
  "cve_id": "CVE-2024-23346",
  "timestamp": "2025-01-14 16:58:33",
  "phases": {
    "primary": {
      "completed": true,
      "timestamp": "2025-01-10 10:40:01"
    },
    "secondary": {
      "completed": true,
      "timestamp": "2025-01-11 20:55:20"
    },
    "extraction": {
      "completed": true,
      "timestamp": "2025-01-12 00:43:02"
    }
  },
  "urls": [
    {
      "url": "https://github.com/materialsproject/pymatgen/blob/master/pymatgen/symmetry/settings.py#L97C1-L111C108",
      "status": "failed",
      "reason": "Failed to fetch content",
      "timestamp": "2025-01-10 10:39:06"
    },
    {
      "url": "https://github.com/materialsproject/pymatgen/commit/c231cbd3d5147ee920a37b6ee9dd236b376bcf5a",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://github.com/materialsproject/pymatgen/security/advisories/GHSA-vgv8-5cpj-qj2f",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://www.vicarius.io/vsociety/posts/critical-security-flaw-in-pymatgen-library-cve-2024-23346",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    }
  ]
}